File: Bamilo/View/IconTableViewCell.m
//
//  IconTableViewCell.m
//  Bamilo
//
//  Created by Ali saiedifar on 1/29/17.
//  Copyright © 2017 Rocket Internet. All rights reserved.
//

#import "IconTableViewCell.h"

@interface IconTableViewCell()
@property (weak, nonatomic) IBOutlet UIImageView* iconImage;
@end

@implementation IconTableViewCell

- (void)setImageName:(NSString *)imageName {
    self.iconImage.image = [UIImage imageNamed:imageName];
}

#pragma mark - Overrides
+ (NSString *)nibName {
    return @"IconTableViewCell";
}

@end

File: IconTableViewCell.swift
//
//  IconTableViewCell.swift
//  Bamilo
//
//  Created by Ali Saeedifar on 9/16/17.
//  Copyright © 2017 Rocket Internet. All rights reserved.
//

import UIKit

class IconTableViewCell: BaseTableViewCell {
    

    @IBOutlet weak var titleLabel: UILabel!
    @IBOutlet weak var cellImageView: UIImageView!
    
    
    override func awakeFromNib() {
        super.awakeFromNib()
        self.setupView()
    }
    
    private func setupView() {
        self.titleLabel.applyStype(font: Theme.font(kFontVariationRegular, size: 12), color: Theme.color(kColorGray3))
        self.cellImageView.image = #imageLiteral(resourceName: "placeholder_list")
    }
    
    override static func nibName() -> String {
        return AppUtility.getStringFromClass(for: self)!
    }
    
    override func prepareForReuse() {
        self.cellImageView.image = #imageLiteral(resourceName: "placeholder_list")
        self.titleLabel.text = nil
    }
}
